code/modules/mob/living/simple_animal/friendly/drone/inventory.dm

```diff
@@ -112,4 +112,31 @@
 
 
 /mob/living/simple_animal/drone/stripPanelEquip(obj/item/what, mob/who, where)
-	..(what, who, where, 1)
\ No newline at end of file
+	..(what, who, where, 1)
+
+/mob/living/simple_animal/drone/verb/quick_equip()
+	set name = "quick-equip"
+	set hidden = 1
+	if (isdrone(src))
```

Comment:
don't typecast src, it's not needed.

Remove the ```if(isdrone(src))``` and the ```var/mob/living/simple_animal/drone/D = src```, replace all instances of ```D``` with ```src``` or just remove the ```D```s (Both would work, Just removing the ```D```s is preferred)